packages/backend/src/ee/services/ai/agents/titleGenerator.ts

```diff
@@ -0,0 +1,50 @@
+import { CoreMessage, generateObject, LanguageModelV1 } from 'ai';
+import { z } from 'zod';
+
+const TitleSchema = z.object({
```

Comment:
you could have a constant here: `TITLE_MAX_LENGTH_CHARS = 60` and use it in all places here